package.json

```diff
@@ -85,7 +85,7 @@
   "devDependencies": {
     "@commitlint/cli": "^16.0.2",
     "@commitlint/config-conventional": "^16.0.0",
-    "@open-draft/test-server": "^0.4.2",
+    "@open-draft/test-server": "^0.5.1",
```

Comment:
Updating this so that custom handlers are applied _before_ the built-in CORS middleware of `test-server`. This way we can override CORS options, like `Access-Control-Allow-Origin` to more specific values (required when using `withCredentials = true` in XHR).